docs-gen/content/rule_set/instances.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Instances"
+date: 2019-07-31T15:27:36+02:00
+weight: 5
+---
+
+As VSS resembles primarily the physical structure of the vehicle. So
```

Comment:
VSS resembles primarily the physical structure of the vehicle, so ...

Reply:
done